pkg/restapi/issuer/operation/operations.go

```diff
@@ -496,20 +538,12 @@ func (c *Operation) validateAdapterCallback(redirectURL string) error {
 		return errors.New("missing state in http query param")
 	}
 
-	tkn := u.Query().Get(tokenQueryParam)
-	if tkn == "" {
-		return errors.New("missing token in http query param")
-	}
-
-	cred, err := c.store.Get(state)
+	_, err = c.store.Get(state)
 	if err != nil {
 		return fmt.Errorf("invalid state : %s", err)
 	}
 
-	err = c.store.Put(tkn, cred)
-	if err != nil {
-		return fmt.Errorf("store adapter token and userID mapping : %s", err)
-	}
+	// TODO validate token existence for the state
```

Comment:
is there a followup issue for this?

Reply:
yes, i'm going to create it. Note: this is not ready for review yet (marked as wip).

Reply:
https://github.com/trustbloc/edge-sandbox/pull/491/files#diff-1b8efd39a1789cdada8e6d9350d7f050R546